test/util/constants.go

```
@@ -28,8 +28,11 @@ import (
 )
 
 const (
+	// TinyTimeout is meant for negative assertions (should NOT happen).
 	TinyTimeout  = 10 * time.Millisecond
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)